src/msw/headerctrl.cpp

```diff
@@ -386,6 +386,27 @@ void wxHeaderCtrl::DoInsertItem(const wxHeaderColumn& col, unsigned int idx)
     {
         wxLogLastError(wxT("Header_InsertItem()"));
     }
+
+    // Per-column IsResizable() cannot be accurately implemented, the resizing
+    // cursor is per-control, not per-column. Enable it if at least one column
+    // is resizeble.
+    bool hasResizableColumns = false;
+    for ( unsigned n = 0; n < GetColumnCount(); n++ )
+    {
+        const wxHeaderColumn& c = GetColumn(n);
+        if (c.IsShown() && c.IsResizeable())
+        {
+            hasResizableColumns = true;
+            break;
+        }
+    }
+
+    long controlStyle = ::GetWindowLong(GetHwnd(), GWL_STYLE);
```

Comment:
Not really a comment for this review, but we do this `GetWindowLong()/modify/SetWindowLong()` and it would be nice to have a reusable function for this in `wxWindowMSW`.